server/release/__tests__/api.test.ts

```diff
@@ -219,8 +219,8 @@ describe('/api/releases', () => {
 			createDiscussionForPub({
 				pubId: pub.id,
 				userId: rando.id,
-				from: 0,
```

Comment:
Changed since this is not a valid position range (`0` is outside the first block)